doc/mgr/telemetry.rst

```diff
@@ -237,29 +237,38 @@ List all collections with:
 
 Where:
 
-**NAME**: Collection name; prefix indicates the channel the collection belongs to.
+.. glossary:: 
 
-**STATUS**: Indicates whether the collection metrics are reported; this is
-determined by the status (enabled / disabled) of the channel the collection
-belongs to, along with the enrollment status of the collection (whether the user
-is opted-in to this collection).
+        NAME
+                Collection name; prefix indicates the channel the collection
+                belongs to.
 
-**DESC**: General description of the collection.
+        STATUS
+                Indicates whether the collection metrics are reported; this is
+                determined by the status (enabled / disabled) of the channel
+                the collection belongs to, along with the enrollment status of
+                the collection (whether the user is opted-in to this
+                collection).
 
-See the diff between the collections you are enrolled to, and the new,
-available collections with:
+        DESC
+                General description of the collection.
+
+To print the diff that displays the differences between (1) the collections you
```

Comment:
To display any differences between the currently-enrolled collections and any new or available collections, run